### DISCUSSION TITLE
Default display order

### DISCUSSION THREAD

**Discussion by peroxovy** [Power-Components/livewire-powergrid#923] [Q&A]
Is there possible to display in other order than ASC data in Data Table?

For example I've got an datasouce like this:

`return Absence::with(['deputy', 'reasons'])->where('user_id', Auth::user()->id)->orderBy('id', 'DESC');`

Which by default will disable sorting by columns. Is it possible to display data in reverese order an then enable filtering and sorting in other order by clicking on that arrows?

So I after I set the datasource method return with `orderBy()` clause, the sorting possibility is no longer available... 
Whet it's without `orderBy()` it works good, but I need by default to have data sorted descending while presenting it to the user.

**Comment by luanfreitasdev** [comment]
Hello, try using this: 

https://v4.livewire-powergrid.com/table/component-settings.html#sort-by-field-and-direction

**Reply by peroxovy** [reply]
Is it present inside 3.x version or only inside newest 4.x?
